README.md

```diff
@@ -1,64 +1,57 @@
 # Code Review During Development
 
-This repository is automatically built from [markdown](https://commonmark.org/) files by [Hugo](https://gohugo.io/) using [GitHub Actions](https://github.com/features/actions).
-This site uses the Hugo [LoveIt](https://hugoloveit.com/) theme.
+This repository is automatically built from [markdown](https://commonmark.org/) files by [Sphinx](https://www.sphinx-doc.org/) and the [MyST parser](https://myst-parser.readthedocs.io)
+using [ReadTheDocs](https://readthedocs.org/).
+This site uses the [Sphinx Book](https://sphinx-book-theme.readthedocs.io) theme.
 This site is available here: https://researchcodereviewcommunity.github.io/dev-review/
```

Comment:
This will not be the URL.

I think it will be https://dev-review.readthedocs.io/en/latest/ but that currently builds a "stub" site.  
I raised a separate issue to fix this link in the README after we merge this, in #78